liblangutil/ErrorReporter.h

```diff
@@ -87,6 +90,23 @@ class ErrorReporter
 
 	void typeError(SourceLocation const& _location, std::string const& _description);
 
+	template <typename... Strings>
```

Comment:
The type itself is a single string, right?

Reply:
I am not sure what you mean by that..